src/components/dishItems/components/menu/Menu.js

```diff
@@ -0,0 +1,39 @@
+// prop-types
+import PropTypes from "prop-types";
+
+// lodash
+import _map from "lodash/map";
+
+// components
+import MenuSection from "./components/menuSection";
+
+// css
+import "./menu.css";
+
+const renderMenuSection = (itemValuesList, itemKey) => {
+  return (
+    <MenuSection key={itemKey} itemList={itemValuesList} heading={itemKey} />
+  );
+};
+
+const Menu = function (props) {
+  const { items, className } = props;
+
+  return (
+    <div className={`menu-items ${className}`}>
+      {_map(items, renderMenuSection)}
+    </div>
+  );
+};
+
+Menu.defaultProps = {
+  items: {},
+  className: "",
+};
+
+Menu.propTypes = {
+  items: PropTypes.objectOf(PropTypes.arrayOf(PropTypes.object)),
```

Comment:
i think Proptypes.objectOf checks the type of Object Properties and not object values. try to check with docs
https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes

Reply:
// An object with property values of a certain type
  optionalObjectOf: PropTypes.objectOf(PropTypes.number),